plugins/user/joomla/src/Extension/Joomla.php

```diff
@@ -172,21 +164,24 @@ public function onUserAfterSave($user, $isnew, $success, $msg): void
             return;
         }
 
+        $app           = $this->getApplication();
+        $language      = $app->getLanguage();
+        $defaultLocale = $language->getTag();
+
         // @todo: Suck in the frontend registration emails here as well. Job for a rainy day.
         // The method check here ensures that if running as a CLI Application we don't get any errors
-        if (method_exists($this->app, 'isClient') && ($this->app->isClient('site') || $this->app->isClient('cli'))) {
+        if (method_exists($app, 'isClient') && ($app->isClient('site') || $app->isClient('cli'))) {
             return;
         }
 
         // Check if we have a sensible from email address, if not bail out as mail would not be sent anyway
-        if (strpos($this->app->get('mailfrom'), '@') === false) {
-            $this->app->enqueueMessage(Text::_('JERROR_SENDING_EMAIL'), 'warning');
+        if (strpos($app->get('mailfrom'), '@') === false) {
+            $app->enqueueMessage($language->_('JERROR_SENDING_EMAIL'), 'warning');
 
             return;
         }
 
-        $defaultLanguage = Factory::getLanguage();
-        $defaultLocale   = $defaultLanguage->getTag();
+
 
         /**
```

Comment:
```suggestion
        /**
```
remove two (of three) blank lines